libraries/textcomposer/impl/src/main/kotlin/io/element/android/libraries/textcomposer/mentions/ResolvedSuggestion.kt

```diff
@@ -17,10 +17,13 @@
 package io.element.android.libraries.textcomposer.mentions
 
 import androidx.compose.runtime.Immutable
+import io.element.android.libraries.matrix.api.core.RoomAlias
 import io.element.android.libraries.matrix.api.room.RoomMember
+import io.element.android.libraries.matrix.api.roomlist.RoomSummary
 
 @Immutable
-sealed interface ResolvedMentionSuggestion {
-    data object AtRoom : ResolvedMentionSuggestion
-    data class Member(val roomMember: RoomMember) : ResolvedMentionSuggestion
+sealed interface ResolvedSuggestion {
+    data object AtRoom : ResolvedSuggestion
+    data class Member(val roomMember: RoomMember) : ResolvedSuggestion
+    data class Alias(val roomAlias: RoomAlias, val roomSummary: RoomSummary) : ResolvedSuggestion
```

Comment:
I prefer to avoid the name clash with the existing `RoomAlias` type. Also `Alias` (which contains a `RoomAlias` is consistent with `Member` (which contains a `RoomMember`).